cvmfs/mountpoint.cc

```diff
@@ -1017,26 +1017,28 @@ bool MountPoint::CheckBlacklists() {
   string blacklist;
   if (!options_mgr_->GetValue("CVMFS_BLACKLIST", &blacklist))
     blacklist = kDefaultBlacklist;
+  blacklist_paths_.push_back(blacklist);
```

Comment:
This assumes that blacklist_paths_ is empty and that CheckBlacklists() will only ever be called once (as is the case now, but maybe that will change one day).  Maybe it should not append here, but instead overwrite, just as the "append" boolean does for the contents of the blacklists.